_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
 
 #### Sickness
 
-If you are sick, you must let us know by 10:00, or as soon as is reasonably
-practical. Do not come to the office. If you come to the office when sick, you
-may be sent home again.
-
-If you're sick but able to work, you can work from home. Everyone at dxw should
-have a laptop, and should generally take it home with them in case it's needed.
-If you're sick and at home but don't have your laptop, it can be couriered to
-you.
-
-When you return to work, you must check that your sickness has been correctly
-recorded in BreatheHR, and update it if necessary. If you're sick for more than
-a week, you'll also need to provide a
-[fit note](https://www.gov.uk/government/publications/the-fit-note-a-guide-for-patients-and-employees).
-
-If you're sick on more than 10 occasions or for more than 10 days over a
-12-month period, we'll invite you to an absence meeting to discuss your
-situation. We may agree with you an action plan and/or a review period. We may
-seek medical advice. Should your attendance fail to improve after a three-month
-period, we will invite you to a further absence meeting to discuss your
-situation. In this meeting we will also give consideration to your possible
-dismissal.
-
-We'll also invite you to an absence meeting if you're off sick for more than
-three weeks on a single occasion. In addition to the above, we will seek to
-support your return to work and discuss with you any temporary or permanent
-adjustments which may assist you. If you are unable to return to work in your
-current role we will consider if there is a suitable alternative role. If we are
-unable to reach an agreement that will permit you to return to work, we will
-give consideration to your possible dismissal.
-
-We will always only consider dismissal after we have exhausted all other
-reasonable options.
+This policy covers both short and longer term sickness absences. Our aim is to
+look after the health and wellbeing of all dxw staff and the needs of the
+company.
+
+##### Communication of sickness
+
+If you’re sick, you must let your line manager or a member of the HR team know
+by 10:00am or as soon as you reasonably can after that. This can be done via
```

Comment:
I think this has got to be "and" rather than "or". If I'm sick and I message my line manager, who is also sick, the message goes into the void and it looks like I was just awol. This is one of the things that's really good about #dxw-sickness, as it's already public, so it doesn't matter who's there to pay attention to it.